### PR TITLE
UI flash message change

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f8582041aaa779848648979472c76a34c7b6d7e8:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/services/test_generic_service_catalogs.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def test_bundles_in_bundle(catalog_item):
 def test_delete_dialog_before_parent_item(catalog_item):
     service_dialog = ServiceDialog(label=catalog_item.dialog)
     service_dialog.delete()
-    flash.assert_message_match(("Dialog \"{}\": Error during 'destroy': Dialog cannot be deleted " +
+    flash.assert_message_match(("Dialog \"{}\": Error during delete: Dialog cannot be deleted " +
```
Comment (psav) [inline cfme/tests/services/test_generic_service_catalogs.py]: This has changed on 55 too?

Comment (psav) [inline cfme/tests/services/test_generic_service_catalogs.py]: Looking at PRTs results it looks like it has not, this will need version picking

**Comment by dajoRH** [comment]
Lint report for commit b792e7e4c7d12e2243d741d0770597b82f9c02b5:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f205faa11e78f250f266c87c51ca689c240d1a40:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
Just waiting PRT....seems like there was an appliance issue, we should retest when that is fixed.

**Pull request merged**